client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -0,0 +1,390 @@
+import styled from '@emotion/styled';
+import { translate, useTranslate } from 'i18n-calypso';
+import { ChangeEvent, useCallback, useState } from 'react';
+import FormSelect from 'calypso/components/forms/form-select';
+import FormInput from 'calypso/components/forms/form-text-input';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { ConfirmationModal } from '../confirmation-modal';
+import SyncOptionsPanel, { CheckboxOptionItem } from '../sync-options-panel';
+
+const synchronizationOptions: CheckboxOptionItem[] = [
+	{
+		name: 'sqls',
+		label: 'Site Database (SQL)',
+		subTitle: translate(
+			'Overwrite the database, posts, pages, products, and orders with staging data.'
+		),
+		checked: false,
+		isDangerous: true,
+	},
+	{
+		name: 'themes',
+		label: translate( 'Themes' ),
+		checked: false,
+		isDangerous: false,
+	},
+	{
+		name: 'plugins',
+		label: translate( 'Plugins' ),
+		checked: false,
+		isDangerous: false,
+	},
+	{
+		name: 'uploads',
+		label: translate( 'Media Uploads' ),
+		checked: false,
+		isDangerous: false,
+	},
+	{
+		name: 'contents',
+		label: translate( 'wp-content directory' ),
+		subTitle: translate( 'excluding themes, plugins, and uploads' ),
+		checked: false,
+		isDangerous: false,
+	},
+	{
+		name: 'roots',
+		label: translate( 'WordPress roots' ),
+		subTitle: translate( 'includes wp-config php and any non WordPress files' ),
```

Comment:
@wojtekn Does the `wp-config.php` file actually get synced over? Doesn't this break the site?

Reply:
I've just tested it and it looks like it's not. I checked restore log and found this entry:

```
Skipping wp-config.php because the restore source is differente: f5:/wp-config.php
```

It would be that it doesn't synchronize `wp-config.php` if we restore from one site to another.

Reply:
What about this one @wojtekn ?
Maybe change that to `includes any non WordPress files`?

Reply:
UPDATE: Just saw your proposal below. Updated.